src/lib/features/export-import-toggles/export-import.e2e.test.ts

```diff
@@ -86,6 +94,19 @@ const createToggle = async (
     );
 };
 
+const addFeatureDependency = async (
```

Comment:
Please use `await app.addDependency(child, parent);` instead since this code is already encapsulated there. You can check feature-toggles.e2e.test.ts to see how it's used